engines/query-sparql/package.json

```diff
@@ -151,6 +149,7 @@
     "@comunica/actor-rdf-update-quads-hypermedia": "^2.0.0",
     "@comunica/actor-rdf-update-quads-rdfjs-store": "^2.0.0",
     "@comunica/bus-http-invalidate": "^2.0.0",
+    "@comunica/bus-rdf-dereference": "^2.0.0",
```

Comment:
Are all required packages in here? You might need to invoke the Comunica packager script again.